sarasota/scripts/layoutMapping.js

```
@@ -17,124 +17,401 @@ function extractPropertyId($) {
 }
 
-function createLayout(spaceType, index, size, yearBuilt, isExterior = false) {
+function baseLayoutDefaults() {
   return {
-    space_type: spaceType,
-    space_index: index,
+    // Required fields per schema (allowing null where permitted)
     flooring_material_type: null,
-    size_square_feet: size,
-    floor_level: isExterior ? null : "1st Floor",
+    size_square_feet: null,
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)